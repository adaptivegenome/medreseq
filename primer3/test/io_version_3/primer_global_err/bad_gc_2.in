PRIMER_SEQUENCE_ID=bad_gc_2
SEQUENCE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_MIN_GC=101
=
