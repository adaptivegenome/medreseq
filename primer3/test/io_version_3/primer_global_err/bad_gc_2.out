PRIMER_SEQUENCE_ID=bad_gc_2
SEQUENCE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_MIN_GC=101
PRIMER_ERROR=Illegal value for PRIMER_MAX_GC and PRIMER_MIN_GC
=
